apps/omg_child_chain/lib/child_chain.ex

```diff
@@ -67,7 +67,7 @@ defmodule OMG.ChildChain do
   defp is_supported(%Transaction.Recovered{}), do: :ok
 
   defp result_with_logging(result) do
-    _ = Logger.debug(" resulted with #{inspect(result)}")
+    _ = Logger.info(" resulted with #{inspect(result)}")
```

Comment:
is this the only occurance of the call? https://github.com/omgnetwork/elixir-omg/pull/1545/files#diff-6a1bd5749a6c9a2d4f25c09e137226a4R61

otherwise you don't really know what the result is from